src/init.ts

```diff
@@ -93,6 +119,9 @@ export function init (options: InitOptions) {
         return upgrade(options)
       }
 
+      let config = DEFAULT_CONFIG
+      config.name = guessProjectName(options)
```

Comment:
This mutates `DEFAULT_CONFIG`. In general, you can't actually run into an issue here, but it's nice to not mutate defaults (so I use `xtend` a lot).